### ISSUE TITLE
Questions about the source of T2T-CHM13 genetic map

### ISSUE THREAD

**Issue by lsaeric** [marbl/CHM13#96]
I noticed that the T2T-CHM13 database does not list the genetic map for the T2T-CHM13 version. Is there any resource available to obtain the genetic map for the T2T-CHM13?

**Comment by arangrhie** [comment]
Hi @lsaeric , we have a [cytoband annotation](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/chm13v2.0_cytobands_allchrs.bed). Is this what you are looking for?

**Comment by lsaeric** [comment]
No, the genetic map I need is that can be used for calculating recombination rate.

> Hi @lsaeric , we have a [cytoband annotation](https://s3-us-west-2.amazonaws.com/human-pangenomics/T2T/CHM13/assemblies/annotation/chm13v2.0_cytobands_allchrs.bed). Is this what you are looking for?

**Comment by arangrhie** [comment]
Sorry, then I am not aware of it..! Do you have the genetic map for hg38? I could try lift that over to CHM13 if that'll be useful.

**Comment by lsaeric** [comment]
> Sorry, then I am not aware of it..! Do you have the genetic map for hg38? I could try lift that over to CHM13 if that'll be useful.

Thank you, and you meant that there is not an available version of genetic map for T2T-CHM13 yet. Additionaly, the best approach to get the resource now is to liftover GRCh38 data to T2T-CHM13 coordinates?

**Comment by arangrhie** [comment]
Yes I am not aware of any genetic map for T2T-CHM13.

>Additionaly, the best approach to get the resource now is to liftover GRCh38 data to T2T-CHM13 coordinates?
I believe so, although it may fall apart at some corner cases.

**Comment by arangrhie** [comment]
To those interested to follow up on this: https://pubmed.ncbi.nlm.nih.gov/40060455/
